starter-code/src/clue.js

```diff
@@ -85,10 +85,90 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+    "mrGreen",
+    "drOrchid",
+    "profPlum",
+    "missScarlet",
+    "mrsPeacock",
+    "mrMustard",
+];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+    "Dinning Room",
+    "Conservatory",
+    "Kitchen",
+    "Study",
+    "Library",
+    "Billiard Room",
+    "Lounge",
+    "Ballroom",
+    "Hall",
+    "Spa",
+    "Living Room",
+    "Observatory",
+    "Theater",
+    "Guest House",
+    "Patio",
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+    "rope",
+    "knife",
+    "candlestick",
+    "dumbbell",
+    "poison",
+    "axe",
+    "bat",
+    "trophy",
+    "pistol",
+];
+
+// declare characters' variables
+var mrGreen = { first_name: "Jacob", last_name: "Green", color: "green", description:  "He has a lot of connections", age:"45", image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", occupation: "Entrepreneur"};
+var drOrchid = { first_name: "Doctor", last_name: "Orchid", color: "while", description:  "PhD in plant toxicology. Adopted daughter of Mr. Boddy", age:"26", image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg", occupation: "Scientist"};
+var profPlum = { first_name: "Victor", last_name: "Plum", color: "purple", description:  "Billionare video game designer", age:"22", image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg", occupation: "Designer"};
+var missScarlet = { first_name: "Kasandra", last_name: "Scarlet", color: "red", description:  "She is an A-list movie star with a dark past", age:"31", image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg", occupation: "Actor"};
+var mrsPeacock = { first_name: "Eleanor", last_name: "Peacock", color: "blue", description:  "She is from a wealthy family and uses her status and money to earn popularity", age:"36", image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg", occupation: "Socialité"};
+var mrMustard = { first_name: "Jack", last_name: "Mustard", color: "yellow", description:  "He is a former football player who tries to get by on his former glory", age:"62", image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg", occupation: "Retired Football player"};
+
+// declare weapons' variables
+var rope = {name: "rope", weight:"10"};
+var knife = {name : "knife", weight:"8"};
+var candlestick = {name: "candlestick", weight:"2"};
+var dumbbell = {name: "dumbell", weight:"30"};
+var poison = {name : "poisons",weight:"2"};
+var axe = {name : "axe", weight:"15"};
+var bat = {name : "bat", weight:"13"};
+var trophy = {name : "trophy" , weight:"25"};
+var pistol = {name : "pistol",weight:"20"};
+
+// random select
+function randomSelector(charactersArray){
```

Comment:
Instead of rewriting the same function like this, keep you code DRY ;) 
You can use it like this  : 

```js 

cardStacks = [];
cardStacks.push(charactersArray, weaponsArray, roomsArray);

//This function randomly selects a card from a card stack
function randomSelector(cardStack){
  var randomCard = cardStack[Math.floor(Math.random()*cardStack.length)];
  return randomCard;
}

```

Basically you are creating an array, that will contain all the other arrays (an array of array :) ) and then you apply a random method to each of the element (forEach method) it will take randomly an element from all the array so you will be able to create your mystery